docker-compose.yml

```diff
@@ -12,8 +12,10 @@ services:
     build: ./back-end
     ports:
       - 3000:3000
+    env_file:
+      - .env
     environment:
-      DB_URL: mongodb://127.0.0.1:27017/todo/tasks
+      DB_URL: DB_URL
```

Comment:
Since you have an env file, you don't need to specify `DB_URL` here

furthermore, you don't even have a `DB_URL` env variable set, _(you have `DB_URI`, which is the better naming btw as the mongoose documentation also calls it URI, kudos for that)_ so the `environment` section is completely unused and therefore unnecessary